### ISSUE TITLE
Docker image tag 'latest' pointing to 2.23.0

### ISSUE THREAD

**Issue by tenjaa** [nulldriver/cf-cli-resource#95]
Hi,
we had some confusion today as the latest tag does not point to the latest version.
https://hub.docker.com/layers/nulldriver/cf-cli-resource/latest/images/sha256-70f008cd97a6eff71c05f69860a90c5ebb0a9c5e113d55071896f3d8df41fb62?context=explore
Do you intend to update the tag?

**Comment by patrickcrocker** [comment]
very strange... the pipeline should have updated the `latest` tag with the `v2.24.0` image.  I'll have to investigate this...

**Comment by patrickcrocker** [comment]
I manually fixed this by tagging v2.24.0 as latest:
```sh
$ docker images
REPOSITORY                   TAG       IMAGE ID       CREATED      SIZE
nulldriver/cf-cli-resource   2.24.0    7e9b48504803   2 days ago   68.3MB
nulldriver/cf-cli-resource   latest    7e9b48504803   2 days ago   68.3MB
```

For the pipeline, it used to work where it would publish the actual and `latest` versions to docker hub... I'm not sure what changed in Concourse over the past year (the last time I published was over a year ago!) but obviously the old behavior wasn't working with the latest version of Concourse (I'm on v7.6.0).  So, after doing some testing on a test pipeline/docker repo I've got an [updated pipeline](/nulldriver/cf-cli-resource/commit/847cf93c7d44e6f4febae2cfe5d8f65a2bdfde37) that should publish correctly in the future.
